server/basicAuthStatic.js

```diff
@@ -0,0 +1,42 @@
+const _ = require('lodash/fp')
+const reduce = _.reduce.convert({cap: false})
```

Comment:
We have this in futil: https://github.com/smartprocure/futil-js/blob/master/src/conversion.js#L27